lintreview/review.py

```diff
@@ -197,10 +197,13 @@ def publish_status(self, problem_count):
             self.publish_ok_comment()
             state = 'success'
             description = 'No lint errors found.'
-        self._repo.create_status(
-            self._pr.head,
-            state,
-            description)
+        status = self.config.get('PULLREQUEST_STATUS', True)
+        if status:
+            self._repo.create_status(
+                self._pr.head,
+                state,
+                description
+            )
```

Comment:
Could we get a test for the new config option? Asserting that the build status doesn't get updated when the config flag is off would be great.